akka-remote/src/main/protobuf/ContainerFormats.proto

```diff
@@ -50,3 +50,17 @@ message Payload {
   required int32 serializerId = 2;
   optional bytes messageManifest = 4;
 }
+
+message Throwable {
+  required string clazzName = 1;
```

Comment:
Why the z-s, I know that class is reserved in Java, but does `className` cause any problems?